templates/windows/HabService.exe.config.erb

```diff
@@ -2,7 +2,10 @@
 <configuration>
   <appSettings>
     <add key="debug" value="false" />
-    <add key="HAB_FEAT_IGNORE_SIGNALS" value="true" />
+    <add key="maximumFilesize" value="2kb" />
```

Comment:
The mean part of me thinks there should be tests for these settings being correct so it doesn't break in future?

if they are optional you will want if blocks around them

Reply:
These changes were tested in vagrant and AWS but not able to test here yet. I made this change based on what our internal hab team advised. 

these are optional. can you explain what you mean by blocks?

Reply:
are you saying to wrap them like this? 
```!#/bin/ruby
variables(exec_start_options: exec_start_options,
                    bldr_url: new_resource.bldr_url,
                    auth_token: new_resource.auth_token,
                    gateway_auth_token: new_resource.gateway_auth_token)
```

Reply:
nvm I read your suggestion wrong lol. let me make these optional

Reply:
Made the new settings optional as you suggested, I've definitely tested multiple times on 2012r2, 2016 and 2019. I am working on getting that testing in here still.